src/browser/fetcher.rs

```diff
@@ -0,0 +1,322 @@
+use directories::ProjectDirs;
+use failure::{format_err, Error};
+use indicatif::{ProgressBar, ProgressStyle};
+use log::*;
+use reqwest::{self, header::CONTENT_LENGTH};
+use zip;
+
+use std::{
+    fs::{self, File, OpenOptions},
+    io::{self, Write},
+    path::{Path, PathBuf},
+    str::FromStr,
+};
+
+pub const CUR_REV: &str = "634997";
+
+const APP_NAME: &str = "headless-chrome";
+const DEFAULT_HOST: &str = "https://storage.googleapis.com";
+
+const REV_URL: &str =
```

Comment:
I think you forgot a `cfg(target_os="")` here, which causes `REV_URL` to be defined twice on Mac / Windows.

Reply:
Same with PLATFORM